### DISCUSSION TITLE
Music change plugin

### DISCUSSION THREAD

**Discussion by wheniownu** [runelite/runelite#15498] [Q&A] [answered]
It would be sick if we had a plugin that would change all of the ingame music to its lofi/beats version.

**Comment by alowaniak** [comment, marked as answer]
Not 100% sure what you mean with "its lofi/beats" version, but I'm assuming you're referring to [OSRSBeatz](https://www.youtube.com/c/OSRSBeatz)?
The [Music Replacer](https://runelite.net/plugin-hub/show/music-replacer) plugin on the pluginhub can do this.
